specifyweb/frontend/js_src/lib/localization/interactions.ts

```diff
@@ -37,20 +37,20 @@ export const interactionsText = createDictionary({
     'fr-fr': 'Créer {modelName:string}',
     'uk-ua': 'Створити {modelName:string}',
   },
-  missing: {
-    'en-us': 'Missing:',
-    'ru-ru': 'Отсутствует:',
-    'es-es': 'Desaparecidos:',
-    'fr-fr': 'Manquant:',
-    'uk-ua': 'Відсутні:',
-  },
   preparationsNotFound: {
     'en-us': 'No preparations were found.',
     'ru-ru': 'Никаких препаратов не обнаружено.',
     'es-es': 'No se encontraron preparados.',
     'fr-fr': "Aucune préparation n'a été trouvée.",
     'uk-ua': 'Препаратів не виявлено.',
   },
+  preparationsNotFoundFor: {
+    'en-us': 'No preparations were found for the following Catalog Numbers.',
```

Comment:
sorry for not being clear. I meant you need to replace
```
No preparations were found for the following Catalog Numbers.
```
with
```
No preparations were found for the following Catalog Numbers:
```

notice the `:` (colon) at the end instead of period